hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -720,7 +723,11 @@ public PendingKeysDeletion getPendingDeletionKeys(
       String volume, String bucket, String startKey,
       CheckedFunction<KeyValue<String, OmKeyInfo>, Boolean, IOException> filter,
       int count) throws IOException {
+    boolean isDataDistributionEnabled = ozoneManager.getScmInfo().getMetaDataLayoutVersion() >=
```

Comment:
no need any logic here, can be part of ScmBlockLocationProtocolClientSideTranslatorPB where conversion can be done based on protocol type compatibility. This logic can be simple.